### DISCUSSION TITLE
Is it possible to change gltf with webp to just normal image like jpg or png?

### DISCUSSION THREAD

**Discussion by Suprhimp** [donmccurdy/glTF-Transform#906] [Q&A] [answered]
I tried my webp gltf file to png with this command

`gltf-transform png pointLight.glb pointLight2.glb`

but it didn't change anything 

Is this impossible?

this is my sample gltf
[pointLight.glb.zip](https://github.com/donmccurdy/glTF-Transform/files/11220162/pointLight.glb.zip)

**Comment by donmccurdy** [comment, marked as answer]
Hi @Suprhimp! By default the `png` command optimizes compression on textures that are already in the PNG format. To convert other textures you'll need to pass the `--formats` option:

```
gltf-transform png pointLight.glb pointLight2.glb --formats "*"
```

This should work for any input format supported by Sharp (https://sharp.pixelplumbing.com/). KTX2 decompression is not currently supported, see #591.

**Reply by Suprhimp** [reply]
It works well with --formats '*'! Thanks! :)
